### PR TITLE
query exchange rate

### PR DESCRIPTION
**Pull request by lk9100**
retrieve exchange rate, for international merchant.

### CODE REVIEW

File: lib/Payment.js [hunk 1]
```diff
@@ -775,6 +775,28 @@ class Payment {
...
+   * wechat international merchant only
+   * retrieve exchange rate for given foreign currency and date
+   * @param {object} query - visit https://pay.weixin.qq.com/wiki/doc/api/external/jsapi.php?chapter=9_15&index=10 for more info
+   * @return {Promise}
+   */
+  queryexchagerate(query) {
+    return this.simpleRequest(
+      this.paymentUrls.QUERY_EXCHANGE_RATE,
+      {
+        ...query,
```
Comment (JasonBoy) [inline lib/Payment.js]: [object spread properties](https://node.green/#ES2018-features-object-rest-spread-properties-object-spread-properties) only supported on node-v8.3+, causing the CI failed

File: demo/index.js [hunk 2]
```diff
@@ -293,6 +294,23 @@ app.get('/settlements', function (req, res) {
...
 
+app.get('/exchange-rate', (req, res) => {
+  const {
+    fee_type, //string 'USD'
+    date, //string '20180801'
+    // visit https://pay.weixin.qq.com/wiki/doc/api/external/jsapi.php?chapter=9_15&index=10 for more info
+    } = req.query;
+  wx.payment.queryexchagerate({
+    fee_type: fee_type || 'GBP',
+    date: date || utils.simpleDate(new Date(), 'yyyyMMdd'),
```
Comment (JasonBoy) [inline demo/index.js]: the format should be `yyyymmdd` in [dateformat package](https://github.com/felixge/node-dateformat/tree/v2_2), the capital `MM` is for minutes

**Comment by lk9100** [comment]
thanks for checking, have addressed them in latest commits :)

File: lib/Payment.js [hunk 3]
```diff
@@ -775,6 +775,25 @@ class Payment {
       }
     );
   }
+
+  /**
+   * wechat international merchant only
+   * retrieve exchange rate for given foreign currency and date
```
Comment (JasonBoy) [inline lib/Payment.js]: The first letter of the first word, better to change it to Capital(e.g: Retrieve exchange...) to be consistent with other jsdoc

File: lib/Payment.js [hunk 4]
```diff
@@ -775,6 +775,25 @@ class Payment {
...
+   * @param {object} query - visit https://pay.weixin.qq.com/wiki/doc/api/external/jsapi.php?chapter=9_15&index=10 for more info
+   * @return {Promise}
+   */
+  queryexchagerate(query) {
```
Comment (JasonBoy) [inline lib/Payment.js]: The method name should be in `camelCase` to be consistent with other methods

**Comment by JasonBoy** [comment]
Also the `settlementQuery()` method may need to change to `querySettlement()` just like other query methods

**Comment by lk9100** [comment]
makes sense to me, fixing...🔨

**Comment by JasonBoy** [comment]
thx

**Pull request merged**